src/Console/WorkCommand.php

```diff
@@ -31,4 +31,16 @@ class WorkCommand extends BaseWorkCommand
      * @var bool
      */
     protected $hidden = true;
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle()
+    {
+        ignore_user_abort(true);
```

Comment:
This line prevents the worker to being killed if it print to stdout after supervisor dies.